### PR TITLE
Specify exactly what packages are required for building on Ubuntu 18.04

### LINKED ISSUE #31: XQEMU.com Getting Started Instructions for Linux are off.
https://xqemu.com/getting-started/#building-on-gnulinux

When trying to follow the first instruction for building on GNU/Linux using the same distro (Ubuntu 18.04):

First enable deb-src via:

    sudo gedit /etc/apt/sources.list

In this file, uncomment first deb-src line.

I can't find any line in the sources.list file that starts with "deb-src". We should probably rephrase this to at least say what the line is so that people can just add it in themselves if their linux install doesn't already have it commented out.

### PR DESCRIPTION
**Pull request by GXTX**
Some *developers* were getting stuck on the dependency process for building on Ubuntu, this removes the requirement to install *all* QEMU deps and just recommends installing exactly what is required.

This also resolves #31.

### CODE REVIEW

File: docs/developers/building.md [hunk 1]
```diff
@@ -42,18 +42,11 @@ And build using the build script:
...
-
-    sudo apt-get update
-
 Install build deps:
 
-    sudo apt-get build-dep qemu
-    sudo apt-get install git libsdl2-dev libepoxy-dev
+    sudo apt-get update
+    sudo apt-get install git libsdl2-dev libepoxy-dev build-essential \
+    pkg-config zlib1g-dev libsdl2-dev libpixman-1-dev
```
Comment (JayFoxRox) [inline docs/developers/building.md]: `libsdl2-dev` is listed twice.

Comment (JayFoxRox) [inline docs/developers/building.md]: Avoid putting `pkg-config` (a binary name)  at the front of the line - maybe also indent? I was wondering why you were running `pkg-config` after installing packages, until I realized it was a package to be installed.

Comment (GXTX) [reply docs/developers/building.md]: I've fixed your concerns here. RE: indentation I'm following the style of the other entries.

Comment (GXTX) [reply docs/developers/building.md]: My mistake! Fixed.

**Comment by JayFoxRox** [comment]
I also think using QEMU pre-configured dependencies is no longer a good idea - we have far fewer requirements because of our stripped configuration.

Some nitpicking: The ordering of the packages (on all platforms) should be the same; however, I'm fine with this as-is, too.

I'm unable to test these instructions myself. CC @mborgerson 
If there's no test soon, I'll probably still merge it, as we can quickly fix it when someone complains.

**Comment by mborgerson** [comment]
Thanks! This is easier for users to understand.

Tested with fresh Docker Ubuntu 18.04 container. As far as dependencies, this works. There's an issue with Python which I'll fix in the docs in a second.
